pkg/storage/s3/s3.go

```diff
@@ -351,7 +351,7 @@ func (d *driver) RemoveStorage(cr *opapi.ImageRegistry) error {
 		return err
 	}
 	_, err = svc.DeleteBucket(&s3.DeleteBucketInput{
-		Bucket: aws.String(d.Config.Bucket),
+		Bucket: aws.String(cr.Spec.Storage.S3.Bucket),
```

Comment:
should use cr.Status's storage config info?

Reply:
yep, updated